velox/expression/CastExpr-inl.h

```diff
@@ -285,6 +285,27 @@ void CastExpr::castTimestampToDate(
   });
 }
 
+template <typename IntType>
+void CastExpr::castIntegerToTimestamp(
+
+    const SelectivityVector& rows,
+    const BaseVector& input,
+    exec::EvalCtx& context,
+    VectorPtr castResult,
+    const facebook::velox::date::time_zone* timeZone,
+    FlatVector<Timestamp>* resultFlatVector) {
+  applyToSelectedNoThrowLocal(context, rows, castResult, [&](int row) {
+    auto timestamp = Timestamp::fromMillis(
+        static_cast<int64_t>(input.as<SimpleVector<IntType>>()->valueAt(row)) *
```

Comment:
I wonder if the conversion to int64_t is implicit and we may not need static_cast.